canvas_modules/common-canvas/src/common-properties/form/EditorForm.js

```diff
@@ -592,6 +607,7 @@ function _makeEditStyleSubPanel(structureDef, l10nProvider, structureMetadata) {
 	const panel = new ControlPanel(
 		structureDef.name,
 		PanelType.GENERAL,
+		"properties-editstyle-sub-panel",
```

Comment:
Why was this added?

Reply:
This className is added to all sub panel groups in tables, its not required and can be removed if we dont want to introduce another className